EpamWebSiteTestingFramework/Epam.TestAutomation.Pages/PageObjects/Pages/JobDetailsPage.cs

```diff
@@ -0,0 +1,66 @@
+using Epam.TestAutomation.Core.BasePage;
+using Epam.TestAutomation.Core.DriverCreator;
+using Epam.TestAutomation.Core.Elements;
+using Epam.TestAutomation.Core.Utils;
+using OpenQA.Selenium;
+
+namespace Epam.TestAutomation.Pages.PageObjects.Pages;
+
+public class JobDetailsPage : BasePage
+{
+    private JobSearchResultPage _resultPage;
+
+    public TextInput FirstName => new(By.Id(
+        "_content_epam_en_careers_job-listings_job_jcr_content_right-container_form_constructor_applicantFirstName"));
+
+    public TextInput LastName => new(By.Id(
+        "_content_epam_en_careers_job-listings_job_jcr_content_right-container_form_constructor_applicantLastName"));
+
+    public TextInput Email => new(By.Id(
+        "_content_epam_en_careers_job-listings_job_jcr_content_right-container_form_constructor_applicantEmail"));
+
+    public override bool IsOpened()
+    {
+        throw new NotImplementedException();
```

Comment:
plase do not let such methos. If you have abstarct method with not implemented exception it means that you have issues with your layering + structure